addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -122,7 +122,9 @@
                             <!-- ==== Header Left ==== -->
                             <div class="col-12 col-md-4 col-lg-3 position-relative">
                                 <div class="d-flex align-items-end justify-content-around h-100">
-                                    <div t-field="channel.image_1920" t-options='{"widget": "image", "class": "o_wslides_course_pict d-inline-block mb-2 mt-3 my-md-0"}' class="h-100"/>
+                                    <div t-field="channel.image_1920" t-options="{'widget': 'image', 'class' : 'o_wslides_course_pict d-inline-block mb-2 mt-3 my-md-0', 'preview_image': 'image_1024'}" class=" h-100">
+                                        <t t-if="channel.partner_has_new_content" t-call="website_slides.course_card_information"/>
```

Comment:
Unless discussed with @LNAodoo , I wouldn't add this here either (not pretty imho).